boards/STM32F103ZET6-F103ZE-Board.md

```diff
@@ -308,40 +309,48 @@ title: "STM32F103ZET6 - F103ZE board"
 
 ## TFT
 
-| Pin   | Connected to |
-| ----- | ------------ |
-|       | NRST         |
-|       | Ground plane |
-|       | PD9          |
-|       | PD10         |
-|       | PE15         |
-|       | PD8          |
-|       | PE13         |
-|       | PE14         |
-|       | PE11         |
-|       | PE12         |
-|       | PE9          |
-|       | PE10         |
-|       | PE7          |
-|       | PE8          |
-|       | PD0          |
-|       | PD1          |
-|       | PD14         |
-|       | PD15         |
-|       | PD5          |
-|       | PD4          |
-|       | PG12         |
-|       | PG0          |
-|       | PB2          |
-|       | PB1          |
-|       | PF8          |
-|       | PF9          |
-|       | N.C.         |
-|       | N.C.         |
-|       | VBAT         |
-|       | Ground plane |
-|       | +3.3V rail   |
-|       | Ground plane |
+| Pin   | Connected to |Function        |
+| ----- | ------------ |----------------|
```

Comment:
Please keep the header of the new column consistent with the other columns, so please add a space before "Function" and add spaces around the dashes (`| --- |` instead of `|---|`).

Reply:
Will try to better follow the existing formatting.